### DISCUSSION TITLE
How to pin/unpin?

### DISCUSSION THREAD

**Discussion by virtualwatt** [valinet/ExplorerPatcher#409] [Q&A]
What's the way to pin/unpin app icons on the taskbar? There are no such items in the context menu. Also searching in the internet there is couple of solutions in Win11 but they are absent probably due to replaced taskbar...

**Comment by BraINstinct0** [comment]
Could you add a screenshot for this? I'm seeing mine, but maybe we have different settings.
For me, right-clicking shows the pin/unpin as the last entry.
1) Which style(Win10/Win11) taskbar are you using?
2) Version of ExplorerPatcher (to report it as a bug if it is indeed one)

**Reply by valinet** [reply]
It’s not a bug. EP does not remove things from the context menu or functionality in general.

**Comment by virtualwatt** [comment]
Hello!

Here is the screenshot:
[image: image.png]
As you can see here is no "pin/unpin" menu item.

Do you need another kind of screenshot?
I'm using Win10 style as I understand...


С уважением, Виталий. Regards, Vitaliy.
==============================
http://www.google.com/profiles/virtualvat


пт, 19 нояб. 2021 г. в 07:28, Valentin-Gabriel Radu <
***@***.***>:

**Comment by virtualwatt** [comment]
Oh sorry forgot to  mention - I've just installed the latest
version 22000.318.38.2 and the screenshot is made for it.


С уважением, Виталий. Regards, Vitaliy.
==============================
http://www.google.com/profiles/virtualvat


сб, 20 нояб. 2021 г. в 18:54, Vitaliy Tkachenko ***@***.***>:
